### PR TITLE
add database schema and store images as reprojected raster in postgis

### PR DESCRIPTION
**Pull request by marcleerink**
- store plastic mask raster in db with polygon bbox and other metadata. 
- Convert to polygons with pixel values and store in related vector table
- downloading, predicting, reprojecting and vectorizing is all happening in memory

### CODE REVIEW

**Comment by marcleerink** [comment]
@emely3h shall we merge this?

**Comment by emely3h** [comment]
@marcleerink definitely, you have the honour of merging :D

**Review by emely3h — APPROVED** [review]
well done

File: main.py [hunk 1]
```diff
@@ -23,28 +37,80 @@ def image_generator(bbox_list, time_interval, evalscript, maxcc):
...
+    )
+    predictor = ScenePredictor(device="cpu")
 
     for data in data_gen:
-        detector = SegmentationModel.load_from_checkpoint(
-            CHECKPOINTS["unet++1"], map_location="cpu", trust_repo=True
-        )
         for _d in data:
             if _d.content is not None:
-                predictor = ScenePredictor(device="cpu")
```
Comment (emely3h) [inline main.py:44]: Maybe we can add a check in here whether the prediction already exists in the db. I was thinking if you start making predictions for a specific area and it stops in the middle then you'd have to rerun everything, would be faster if we can skip the calculating the predictions in this case? What do you think?
(But I think that would be a new task, I'd first close this pull request)

**Pull request merged**